lib/core-integration/src/Test/Integration/Framework/TestData.hs

```diff
@@ -264,10 +265,16 @@ errMsg403TxTooBig n = "I had to select " ++ show n ++ " inputs to construct the\
     \ Try sending a smaller amount."
 
 errMsg404MalformedTxPayload :: String
-errMsg404MalformedTxPayload = "I couldn't decode the payload that seems to be an\
-    \ externally-signed transaction due to: wrongly constructed binary blob. \
-    \Make sure to send a base64-encoded binary blob, in the proper binary \
-    \format of the already-serialized transaction."
+errMsg404MalformedTxPayload =
+    "I couldn't verify that the payload has the correct binary \
+    \format due to: wrongly constructed binary blob. \
+    \I won't send it to the node. Make sure the blob has the correct binary \
+    \format (and is encoded as hex when sent by json or raw when sent by\
+    \ octet-stream)."
+
+errMsg404WronglyEncodedTxPayload :: String
```

Comment:
Maybe it's just a naming issue here but why is the error `404` for what seems to be an invalid request? The semantic looks off.

Reply:
will change to `400`